test/integration/minishift.go

```diff
@@ -23,10 +23,12 @@ import (
 	"fmt"
 	"github.com/minishift/minishift/test/integration/util"
 	"strings"
+	"sync"
 	"time"
 )
 
-var lastCommandOutput CommandOutput
+var commandOutputs []CommandOutput
```

Comment:
IMO this should be part of the Minishift struct, since the way it is used from within the struct breaks imo encapsulation. But let's move on for now